free-1.md

```diff
@@ -9,42 +9,141 @@ good way to meet their needs.
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
+This language serves the need to take notes in economics classes on the computer.
+My personal experience, and those of a couple other economics students, finds
+that taking notes on the computer allows for a more complete transcript of the
+lessons given by economics professors. Economics professors tend to provide
+powerpoint slides that outline major points -- or worse, major questions -- of
+the lecture and then provide the context or answers to these points verbally.
+Often, the homework questions and exam questions relate to the things _said_ in
+class. Thus, it's extremely helpful to transcribe lectures, in my opinion. 
+Typing on the computer is much faster than writing by hand, so transcribing is
+more possible on the computer.  However, the biggest issue is that economics 
+involves many graphs. Copying down these graphs by hand is currently the main 
+option for note-taking, unless one searches for the exact graph on Google
+and then copies it into ones notes -- highly inconvenient and not fast enough
+for a classroom setting.  Thus, I found that I had my notes in two places: on
+my computer, and loose sheets of paper with graphs. Eventually, I switched note
+taking style to only take notes by hand, but it still feels inefficient to this
+day, and frustrates me immensely when exams come around and questions regard 
+content _said_ in class but I failed to take note of.
+
+Thus, I propose a DSL that is a markup language specifically for economics to
+allow transcribing notes efficiently, but most importantly, allow for graphs to
+be embedded in the notes very quickly and easily.  This allows a economics
+student to have their notes all in one place, and the opportunity to transcribe
+the verbal lecture from the professor while having relevant graphs available for
+reference.  
 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
-
+A DSL is appropriate because lecture notes consists of content written in some
```

Comment:
I think the focus should be on the graphs instead of the note-taking because note-taking doesn't need to be parsed or interpreted in special ways. But yeah, finding a way to express graphs in a clear language can be useful for embedding.